packages/graphing/src/axis/axes.jsx

```diff
@@ -66,16 +68,33 @@ export class RawXAxis extends React.Component {
   render() {
     const { includeArrows, classes, graphProps } = this.props;
     const { scale, domain, size, range } = graphProps;
-    const columnTicksValues = getTickValues({ ...domain, step: domain.labelStep });
 
     // Having 0 as a number in columnTicksValues does not make 0 to show up
     // so we use this trick, by defining it as a string:
     const tickValues =
       (domain.labelStep || range.labelStep) && domain.min <= 0
-        ? ['0', ...columnTicksValues]
-        : columnTicksValues;
+        ? ['0', ...this.props.columnTicksValues]
+        : this.props.columnTicksValues;
```

Comment:
Instead of this.props.columnTicksValues, get columnTicksValues from the props (line 69).

Reply:
ok